gnofract4d

```diff
@@ -62,11 +62,7 @@ def gtkmain(userConfig, options):
 
     mainWindow = main_window.MainWindow(userConfig)
     mainWindow.apply_options(options)
-
-    if mainWindow.f.thaw():
-        gi.require_version('GLib', '2.0')
-        from gi.repository import GLib
-        GLib.idle_add(mainWindow.on_fractal_change)
```

Comment:
My first attempt used idle_add to call first_draw. Removing it didn't seem to make a difference, but maybe we still want it to keep supporting less powerful CPUs, or just in case there are other demands on the system.

Reply:
Removing idle_add was the cause of the additional draw on thaw() above.